### PR TITLE
add Russian translation

### PR DESCRIPTION
**Pull request by VictorUvarov**
## What does this PR accomplish?

Adds Russian translation

I'm not sure what script you used to generate the translation files, so I didn't bother. Let me know if you need me to add the generated files. 

## Did you add any dependencies?
No

## How did you test the change?
Had my Russian parents double check, they argued about some words, but they both agreed that the translation is fine.

### CODE REVIEW

File: client/flutter/lib/l10n/intl_ru.arb [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-    "infoForParents":"",
-    "childStress":"",
-    "childLove":"",
-    "childDistance":"",
-    "childRoutines":"",
-    "childFacts":""
-  }
\ No newline at end of file
+  "worldHealthOrganization": "Всемирная организация здоровья",
+  "learnMore": "Учить больше",
```
Comment (zualas) [inline client/flutter/lib/l10n/intl_ru.arb]: Узнай больше

File: client/flutter/lib/l10n/intl_ru.arb [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
-    "infoForEveryone":"",
-    "infoForParents":"",
-    "childStress":"",
-    "childLove":"",
-    "childDistance":"",
-    "childRoutines":"",
-    "childFacts":""
-  }
\ No newline at end of file
+  "worldHealthOrganization": "Всемирная организация здоровья",
```
Comment (zualas) [inline client/flutter/lib/l10n/intl_ru.arb]: Всемирная организация здравоохранения

File: client/flutter/lib/l10n/intl_ru.arb [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
+  "learnMore": "Учить больше",
+  "protectYourself": "Защити себя",
+  "checkYourHealth": "Проверь свое здоровье",
+  "localMaps": "Локальные карты",
+  "shareTheApp": "Поделиться приложением",
```
Comment (zualas) [inline client/flutter/lib/l10n/intl_ru.arb]: Поделись приложением

File: client/flutter/lib/l10n/intl_ru.arb [hunk 4]
```diff
@@ -1,23 +1,26 @@
...
+  "feelingDistressed": "Чувствуете себя огорченным?",
+  "washHands": "Часто мойте руки с мылом и проточной водой",
+  "cougningAndSneezing": "При кашле и чихании покройте рот и нос согнутым локтем или тканью",
+  "throwAwayTissue": "Выбрасывайте ткань в закрытую емкость сразу после использования",
+  "washHandsFrequently": "Часто мойте руки",
+  "socialDistancing": "Избегайте тесного контакта и сохраняйте физическое дистанцирование",
+  "seekMedicalCare": "Обратитесь за медицинской помощью немедленно, если у вас жар, кашель и затрудненное дыхание",
+  "infoForEveryone":"Информация для всех",
+  "infoForParents":"Информация для родителей",
+  "childStress":"Дети могут реагировать на стресс по-разному, например, быть более цепкими, тревожными, замкнутыми, рассерженными или взволнованными, ночное недержание мочи и т. д. \nРеагировать на реакции вашего ребенка поддерживающим образом, выслушивать их опасения и уделять им дополнительную любовь и внимание.",
```
Comment (zualas) [inline client/flutter/lib/l10n/intl_ru.arb]: Дети могут реагировать на стресс по-разному, например, быть более навязчивыми, тревожными, замкнутыми, рассерженными или взволнованными, ночное недержание мочи и т. д. \nВоспринимайте реакции вашего ребенка поддерживающим образом, слушайте их опасения и уделяйте им дополнительную любовь и внимание.

File: client/flutter/lib/l10n/intl_ru.arb [hunk 5]
```diff
@@ -1,23 +1,26 @@
...
+  "childDistance":"Старайтесь держать детей рядом с родителями и семьей и старайтесь не разлучать детей и тех, кто за ними ухаживает, насколько это возможно. Если происходит разлучение (например, госпитализация), обеспечьте регулярный контакт (например, по телефону) и поддержку.",
+  "childRoutines":"Соблюдайте как можно больше регулярных процедур и расписаний или помогайте создавать новые в новой среде, включая школу / обучение, а также время для безопасной игры и отдыха. "," факты о ребенке ":" Предоставьте факты о том, что произошло, объясните что происходит сейчас, и дать им четкую информацию о том, как снизить риск заражения этой болезнью словами, которые они могут понять в зависимости от своего возраста. Это также включает в себя предоставление информации о том, что может произойти, с повторным заверением (например, член семьи и / или ребенок могут начать плохо себя чувствовать, и, возможно, им придется некоторое время идти в больницу, чтобы врачи могли помочь им чувствовать себя лучше).",
```
Comment (zualas) [inline client/flutter/lib/l10n/intl_ru.arb]: Придерживайтесь обычного режима насколько возможно или помогайте создавать новый в новой среде, включая школу / обучение, а также время для безопасной игры и отдыха

File: client/flutter/lib/l10n/intl_ru.arb [hunk 6]
```diff
@@ -1,23 +1,26 @@
...
+  "childFacts":"Предоставьте факты о том, что произошло, объясните, что происходит сейчас, и дайте им четкую информацию о том, как снизить риск заражения этой болезнью словами, которые они могут понять в зависимости от своего возраста.\nЭто также включает предоставление информации о том, что может произойти убедительным образом (например, член семьи и / или ребенок могут начать плохо себя чувствовать, и им, возможно, придется на некоторое время пойти в больницу, чтобы врачи могли помочь им чувствовать себя лучше).",
+  "whoMythBusters": "ВОЗ Разрушители мифов",
+  "travelAdvice": "Совет путешествия",
```
Comment (zualas) [inline client/flutter/lib/l10n/intl_ru.arb]: Советы путешественникам

**Comment by sapte91** [comment]
@VictorUvarov Thank you so much for your support of this project and for the time you spent on this PR - we’re so humbled by your willingness to jump in! We have an important update on localization for the application: We’ll primarily be sourcing and managing translations from Lingo24, our translation management system. Also, our designs changed, so these translations are now out of date. As such, I’m closing this PR.

We will still need translation volunteers - as our set of supported languages and locales grows, there will be plenty of opportunity to contribute! Our projected needs for the next two months are in native speaker LQA and translations for languages that we cannot support on Lingo24. We apologize that we are still in the early process of figuring out how to manage all the contributions and will have more details soon. Until then, please fill out the volunteer form below and we’ll get back in touch shortly.

https://forms.gle/DTqvaqh4mE7bDe5U7

**Pull request closed without merging**